resources/views/ItemEdit/modal/save.blade.php

```diff
@@ -0,0 +1,62 @@
+<style>
+    .card-modal{
```

Comment:
Same code with above, suggest that it is inside one css file

Reply:
I just fogot to erase  save and delete css.
I'll do it